rental-car-manager/editorextensions/rental-car-manager/src/modal/rentacarmodal.ts

```diff
@@ -218,4 +219,14 @@ export class RentACarModal extends Modal {
       }
     }
   }
+
+  private drawCars(carBBs: Map<Car, Box>, carBlockDef: BlockDefinition) {
+    const page = this.viewport.getCurrentPage();
+    for (const [car, carBB] of carBBs.entries()) {
+      const carBlock = page?.addBlock({
+        ...carBlockDef,
```

Comment:
extreme nit: the spread might feel like magic at first glance, if you don't know what syntax `addBlock` takes in. Consider explicitly specifying, just to make it more accessible to readers?

but also this is effort. And maybe using the spread here is better cuz it encourages readers to understand, and gives them ideas for best practices